.github/workflows/e2e-test-manual.yml

```diff
@@ -35,6 +35,7 @@ on:
           - "verify"
           - "recover"
           - "malicious join"
+          - "self-managed infra"
```

Comment:
I think it would be beneficial to not have this as a test payload, but as a separate test input (New workflow if needed because of input limitations).
Otherwise we are limited to a single test payload for self-managed infrastructure tests.